src/v/cluster/feature_manager.cc

```diff
@@ -310,6 +312,24 @@ ss::future<> feature_manager::maybe_log_license_check_info() {
                 enterprise_features.enabled()));
         }
     }
+
+    if (std::ranges::any_of(
```

Comment:
> nit: it would be nice to make this method less license-specific now that we also log auth-related log messages as well.

@pgellert I'm not sure what the suggestion is here, I think I'm missing some context.

Reply:
This code is inside the method `feature_manager::maybe_log_license_check_info` which has been used specifically for logging the license nag, whereas the new PLAIN-specific error message is unrelated.

My suggestion here is to make the method's name more generic (e.g., `log_periodic_reminders`), extract the part that's license-specific, extract the new code to a new method that has auth/plain in the name, and rename the env-var to something not license-specific (e.g., `__REDPANDA_PERIODIC_REMINDER_INTERVAL_SEC`).

Reply:
Exactly